src/ui/control/geolocate_control.js

```diff
@@ -406,10 +406,23 @@ class GeolocateControl extends Evented {
     }
 
     /**
-     * Trigger a geolocation
-     *
-     * @returns {boolean} Returns `false` if called before control was added to a map, otherwise returns `true`.
-     */
+    * Trigger a geolocation
```

Comment:
```suggestion
    * Programmatically request and move the map to the user's location. 
```

I'm not sure if this is better or worse.